### ISSUE TITLE
Please support a track from mpd is splitted by multi-init.

### ISSUE THREAD

**Issue by CtrlVaGain** [devine-dl/devine#55]
**Is your feature request related to a problem? Please describe.**
a track from mpd is splitted by multi-init, like "id=7 init link startNumber=1 <S t="1992" d="48048" r="149"/>" and "id=7  init link startNumber=151 <S t="7209192" d="48048" r="558"/>".
In this case, only first part can be download.

**Describe the solution you'd like**
Combine these parts with same id and continuous startnumber in a track.

Thanks in advance

**Comment by rlaphoenix** [comment]
I'm not sure what you mean to be quite honest. The DASH parser and downloader is currently limited to only use the first `Period`. Is your manifest spanning across multiple `Period`s? I.e. where the service would insert ad breaks between Periods? Is this what you mean?

**Comment by CtrlVaGain** [comment]
> I'm not sure what you mean to be quite honest. The DASH parser and downloader is currently limited to only use the first `Period`. Is your manifest spanning across multiple `Period`s? I.e. where the service would insert ad breaks between Periods? Is this what you mean?
> 
> An example MPD would likely be best.

Yes, that's exactly what i mean, and do you have a email or something, i would like to sent mpd privately

**Comment by rlaphoenix** [comment]
Well, there's a problem with supporting multi-period manifests. While it might or might not be the case for your manifest, most manifests do not have any way to link one Representation (or even Adaptation Set) between one Period and another. There might not even be the same bitrate's available, or even the same resolution, across periods.

There isn't usually a reliable way to link representations to their correspondents. Some manifests have literally no identifiers to go by at all either. It's a difficult task to do.

**Comment by CtrlVaGain** [comment]
> Well, there's a problem with supporting multi-period manifests. While it might or might not be the case for your manifest, most manifests do not have any way to link one Representation (or even Adaptation Set) between one Period and another. There might not even be the same bitrate's available, or even the same resolution, across periods.
> 
> There isn't usually a reliable way to link representations to their correspondents. Some manifests have literally no identifiers to go by at all either. It's a difficult task to do.

ok, not in a hurry, and nm3u8-re can completely download the track across multiple period, maybe can refer to the source code.
